integration/setup/common.go

```diff
@@ -85,17 +85,6 @@ func SkipForPostgresWithReason(tb testing.TB, reason string) {
 	}
 }
 
-// SkipForMongoWithReason skips the current test for mongodb handler.
-//
-// Ideally, this function should not be used. It is allowed to use it in Tigris-specific tests only.
-func SkipForMongoWithReason(tb testing.TB, reason string) {
-	tb.Helper()
-
-	if *handlerF == "mongodb" {
```

Comment:
`-handler` flag is defined as "handler to use for in-process FerretDB". It never has "mongodb" value.